appveyor.yml

```diff
@@ -5,6 +5,6 @@ build_script:
 - cmd: >-
     mkdir build &&
     cd build &&
-    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON &&
+    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/W0 /EHsc" &&
```

Comment:
Did you consider other levels that do not suppress all warnings?  Is the number of warnings out of control?

Reply:
The [documentation](https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-160) says that `/W1` is the default for the command-line compiler.
Without this pull request, we get around 30,000 lines of compiler output in a build and with this patch, we are down to less than 1,000 lines.